IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicates the cross port capacity of the distribution frame.
```

Comment:
Is this completely clear (PortCapacity : IfcInteger)? Excuse my ignorance.

Reply:
Updated the documentation: https://github.com/bSI-InfraRoom/IFC-Specification/pull/173/commits/4ed676f1d10748cf1dc4a55f7049ef2ff53de67b

Reply:
@larswik is this resolved in your view?